docs/topics/db/aggregation.txt

```diff
@@ -643,6 +643,54 @@ aggregate that author count, referencing the annotation field:
     >>> Book.objects.annotate(num_authors=Count("authors")).aggregate(Avg("num_authors"))
     {'num_authors__avg': 1.66}
 
+When you are generating an aggregate by using ``.values()`` clause to create the
+grouping for the annotation on MySQL, and the ``ONLY_FULL_GROUP_BY`` SQL mode
+is enabled, it may be necessary to apply :class:`~django.db.models.AnyValue` if
+the annotation includes a mix of aggregated and non-aggregated functions.
```

Comment:
I think we usually use `aggregate` and not `aggregated` to refer to such functions.

```suggestion
the annotation includes a mix of aggregate and non-aggregate expressions.
```